Chinese/little_prince/this_is_a_story_about_a_boy.md

```diff
@@ -0,0 +1 @@
+The Little Prince is a play based on the book of the same name by Antoine de Saint-Exupéry, adapted by Rick Cummins and John Scoullar before 2000. Rick Cummins wrote the music, and John Scoullar wrote the script and lyrics. There are several changes from the book, including the omission of the drunkard (tippler), switchman and merchant characters; the removal of a great deal of the narration from the aviator; significant changes to the rose scenes; and a large change in the order of events.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.